Snippets/scale-font.py

```diff
@@ -0,0 +1,273 @@
+                            class2Record.Value1.XPlacement = scaleValue_toUPM(v, UPM, target_UPM)
+
+
+def scaleLookupType3(subTable, UPM, target_UPM):
+    for entryExitRecord in subTable.EntryExitRecord:
+        entryExitRecord.EntryAnchor.XCoordinate = scaleValue_toUPM(entryExitRecord.EntryAnchor.XCoordinate, UPM, target_UPM)
+        entryExitRecord.EntryAnchor.YCoordinate = scaleValue_toUPM(entryExitRecord.EntryAnchor.YCoordinate, UPM, target_UPM)
+        entryExitRecord.ExitAnchor.XCoordinate = scaleValue_toUPM(entryExitRecord.ExitAnchor.XCoordinate, UPM, target_UPM)
+        entryExitRecord.ExitAnchor.YCoordinate = scaleValue_toUPM(entryExitRecord.ExitAnchor.YCoordinate, UPM, target_UPM)
+
+def scaleLookupType4(subTable, UPM, target_UPM):
+    for baseRecord in subTable.BaseArray.BaseRecord:
+        for baseAnchor in baseRecord.BaseAnchor:
+            if baseAnchor:
+                baseAnchor.XCoordinate = scaleValue_toUPM(baseAnchor.XCoordinate, UPM, target_UPM)
+                baseAnchor.YCoordinate = scaleValue_toUPM(baseAnchor.YCoordinate, UPM, target_UPM)
+    for markRecord in subTable.MarkArray.MarkRecord:
+        markRecord.MarkAnchor.XCoordinate = scaleValue_toUPM(markRecord.MarkAnchor.XCoordinate, UPM, target_UPM)
+        markRecord.MarkAnchor.YCoordinate = scaleValue_toUPM(markRecord.MarkAnchor.YCoordinate, UPM, target_UPM)
+
+def scaleLookupType5(subTable, UPM, target_UPM):    
+    for markRecord in subTable.MarkArray.MarkRecord:     
+        markRecord.MarkAnchor.XCoordinate = scaleValue_toUPM(markRecord.MarkAnchor.XCoordinate, UPM, target_UPM)
+        markRecord.MarkAnchor.YCoordinate = scaleValue_toUPM(markRecord.MarkAnchor.YCoordinate, UPM, target_UPM)
+    for ligatureAttach in subTable.LigatureArray.LigatureAttach:
+        for componentRecord in ligatureAttach.ComponentRecord:
+            for ligatureAnchor in componentRecord.LigatureAnchor:
+                ligatureAnchor.XCoordinate = scaleValue_toUPM(ligatureAnchor.XCoordinate, UPM, target_UPM)
+                ligatureAnchor.YCoordinate = scaleValue_toUPM(ligatureAnchor.YCoordinate, UPM, target_UPM)
+                
+def scaleLookupType6(subTable, UPM, target_UPM): 
+    for mark1Record in subTable.Mark1Array.MarkRecord:
+        mark1Record.MarkAnchor.XCoordinate = scaleValue_toUPM(mark1Record.MarkAnchor.XCoordinate, UPM, target_UPM)
+        mark1Record.MarkAnchor.YCoordinate = scaleValue_toUPM(mark1Record.MarkAnchor.YCoordinate, UPM, target_UPM)
+    for mark2Record in subTable.Mark2Array.Mark2Record:
+        for mark2Anchor in mark2Record.Mark2Anchor:
+            mark2Anchor.XCoordinate = scaleValue_toUPM(mark2Anchor.XCoordinate, UPM, target_UPM)
+            mark2Anchor.YCoordinate = scaleValue_toUPM(mark2Anchor.YCoordinate, UPM, target_UPM)
+
+def scaleLookupType7(subTable, UPM, target_UPM):
+    pass 
+    
+def scaleLookupType8(subTable, UPM, target_UPM):
+    pass        
+
+def scaleFont(tt, target_UPM):
```

Comment:
`tt` is not the most readable name for a variable that holds an instance of TTFont. Use `font` or `ttFont`

Reply:
okay!